exercises/say/say_test.rb

```diff
@@ -1,128 +1,116 @@
   end
 
-  def test_14
+  def test_fourteen
     skip
-    assert_equal 'fourteen', Say.new(14).in_english
+    question = '14'
+    assert_equal(fourteen, SayProblem.new(question).answer)
   end
 
   def test_twenty
     skip
-    # This really shouldn't be twenty-zero
-    assert_equal 'twenty', Say.new(20).in_english
+    question = '20'
+    assert_equal(twenty, SayProblem.new(question).answer)
   end
 
-  def test_twenty_two
+  def test_twenty-two
     skip
-    assert_equal 'twenty-two', Say.new(22).in_english
+    question = '22'
+    assert_equal(twenty-two, SayProblem.new(question).answer)
   end
 
-  def test_100
+  def test_one_hundred
     skip
-    assert_equal 'one hundred', Say.new(100).in_english
+    question = '100'
+    assert_equal(one hundred, SayProblem.new(question).answer)
   end
 
-  def test_120
+  def test_one_hundred_twenty-three
     skip
-    assert_equal 'one hundred twenty', Say.new(120).in_english
+    question = '123'
+    assert_equal(one hundred twenty-three, SayProblem.new(question).answer)
   end
 
-  def test_123
+  def test_one_thousand
     skip
-    assert_equal 'one hundred twenty-three', Say.new(123).in_english
+    question = '1000'
+    assert_equal(one thousand, SayProblem.new(question).answer)
   end
 
-  def test_1_thousand
+  def test_one_thousand_two_hundred_thirty-four
     skip
-    assert_equal 'one thousand', Say.new(1000).in_english
+    question = '1234'
+    assert_equal(one thousand two hundred thirty-four, SayProblem.new(question).answer)
   end
 
-  def test_1_thousand_234
+  def test_one_million
     skip
-    expected = 'one thousand two hundred thirty-four'
-    assert_equal expected, Say.new(1234).in_english
+    question = '1000000'
+    assert_equal(one million, SayProblem.new(question).answer)
   end
 
-  def test_1_million
+  def test_one_million_two_thousand_three_hundred_forty-five
     skip
-    assert_equal 'one million', Say.new(10**6).in_english
+    question = '1002345'
+    assert_equal(one million two thousand three hundred forty-five, SayProblem.new(question).answer)
   end
 
-  def test_1_million_and_some_crumbs
+  def test_one_billion
     skip
-    assert_equal 'one million two', Say.new(1_000_002).in_english
+    question = '1000000000'
+    assert_equal(one billion, SayProblem.new(question).answer)
   end
 
-  def test_1_million_2_thousand_345
+  def test_a_big_number
     skip
-    expected = 'one million two thousand three hundred forty-five'
-    assert_equal expected, Say.new(1_002_345).in_english
+    question = '987654321123'
+    assert_equal(nine hundred eighty-seven billion six hundred fifty-four million three hundred twenty-one thousand one hundred twenty-three, SayProblem.new(question).answer)
   end
 
-  def test_1_billion
+  def test_numbers_below_zero_are_out_of_range
     skip
-    assert_equal 'one billion', Say.new(10**9).in_english
+    question = '-1'
+    assert_equal(-1, SayProblem.new(question).answer)
```

Comment:
These tests that assert `-1` should actually be testing that an error is raised.

It looks like this is happening because the `error_assertion` isn't being used in your `SayCase` class.